torch/nn/utils/clip_grad.py

```diff
@@ -125,9 +125,12 @@ def _clip_grads_with_norm_(
     The gradients will be scaled by the following calculation
 
     .. math::
-        grad = grad * \frac{max\_norm}{total\_norm + 1e-6}
+        grad = grad * min{ \frac{max\_norm}{total\_norm + 1e-6}, 1}
```

Comment:
You can check the rendering [here](https://docs-preview.pytorch.org/pytorch/pytorch/158200/generated/torch.nn.utils.clip_grads_with_norm_.html#torch.nn.utils.clip_grads_with_norm_), I think this should be

```suggestion
        grad = grad * \min(\frac{max\_norm}{total\_norm + 1e-6}, 1)
```
to render correctly

Reply:
@mikaylagawarecki, required suggested changes done.

cc: @spzala